Sources/ProjectDescription/RunActionOptions.swift

```diff
@@ -8,6 +8,9 @@ public struct RunActionOptions: Equatable, Codable {
     /// Region to use when running the app.
     public var region: String?
 
+    /// Show non localized string as UPPERCASE when running the app.
+    public var showNonLocalizedStringsAsUppercase: Bool
```

Comment:
What is the `AsUppercase` for? I see [XCScheme.LaunchAction](https://github.com/tuist/XcodeProj/commit/2b9c2acfbabc87649d65486b8171c83cbccc18b7#diff-e034a7d7b7a01c4e1b3d83eb419ecf1e9c6cfb76e7fe9ed2e812222f4149c572R68) calls it `showNonLocalizedStrings`, so I'd stick to that naming.

Reply:
this was a suggestion by @danyf90 
I'll revert the change

Reply:
What's the effect of `showNonLocalizedStrings` in the project? I was in favor of consistency, but if the name is not representative enough, the `AsUppercase` bit might make sense.

Reply:
on the project nothing, but when running the app, strings that are not localized are UPPERCASE, to be easier to spot.

Reply:
IMHO the fact that it's uppercase is an implementation details that only apple can control. if they decide in their next xcode update to make the text pink and blinking, we should not have to change the name of the option

Reply:
Alright, in that case remove the uppercase from the name.